src/Google/autoload.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+$file = __DIR__ . '/../../vendor/autoload.php';
```

Comment:
@kesar this is a deprecated file meant to move users of v1.x over to 2 with minimal difficulty. If a user has a custom `vendor` directory, they should not be using this file anyway, so not think it's worth handling this edge case.

@tmatsuo I am unclear what that would accomplish.